pandas-stubs/core/groupby/generic.pyi

```diff
@@ -143,6 +143,12 @@ class SeriesGroupBy(GroupBy, Generic[S1]):
         legend: bool = ...,
         **kwargs,
     ) -> AxesSubplot: ...
+    def idxmax(
+        self, axis: AxisType = ..., skipna: bool = ..., numeric_only: bool = ...
+    ) -> Series: ...
+    def idxmin(
+        self, axis: AxisType = ..., skipna: bool = ..., numeric_only: bool = ...
```

Comment:
Looking at the code, it turns out that `numeric_only` is not a parameter for `SeriesGroupBy.idxmax()` and `SeriesGroupBy.idxmin()`, so can you remove them?